packages/cojson/src/coValues/coStream.ts

```diff
@@ -306,20 +320,9 @@ export class RawBinaryCoStreamView<
             const chunk = base64URLtoBytes(
                 item.chunk.slice(binary_U_prefixLength),
             );
-            // totalLength += chunk.length;
             chunks.push(chunk);
-
-            if (Date.now() - lastProgressUpdate > 100) {
-                lastProgressUpdate = Date.now();
-            }
         }
 
-        // const after = performance.now();
-        // console.log(
-        //     "getBinaryChunks bandwidth in MB/s",
-        //     (1000 * totalLength) / (after - before) / (1024 * 1024)
-        // );
```

Comment:
Removed this debug code to make it easier for me to understand the logic.

Happy to revert the change if this code is still useful